daemon/daemon_unix.go

```diff
@@ -116,7 +116,7 @@ func getCPUResources(config containertypes.Resources) *specs.CPU {
 		// https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt
```

Comment:
Can you also revert the comments that were changed in the linked PR? (Where relevant)